sos/plugins/kubernetes.py

```diff
@@ -88,6 +88,13 @@ def setup(self):
                 "{} get -o json pv".format(kube_cmd)
```

Comment:
Needs a "[plugin] change description" commit format, Signed-off-by, and there's a little tpyo in the text ('convienacne').